src/main/java/com/epam/jdbc/service/serviceimpl/StudentServiceImpl.java

```diff
@@ -37,7 +41,7 @@ public Student findUserByEmail(String email) {
                 student.setBirthDay(resultSet.getDate("date"));
             }
         } catch (SQLException e) {
-            throw new RuntimeException(e);
+            logger.error("Can not execute query");
         }
```

Comment:
Please, besides logging throw exception in the catch block.

Reply:
Ok

Reply:
Resolved